docs/user-guide/applications.md

```diff
@@ -1,3 +1,133 @@
+
+* **Resource Scanning**: You can scan for vulnerabilities using Devtron's [resource scanning](../user-guide/security-features.md#from-app-details) feature.
+
+* **Single-pane View**: All Argo CD apps will show details such as their app status, environment, cluster, and namespace together in one dashboard. 
+
+* **Feature-rich Options**: Clicking an Argo CD app will give you access to its logs, terminal, events, manifest, available resource kinds, pod restart log, and many more.
+
+{% hint style="info" %}
+### Additional References
+[ArgoCD: Standalone Configuration vs Devtron Configuration](https://devtron.ai/blog/argocd-standalone-configuration-vs-devtron-configuration/#argocd-installation-and-configuration)
+{% endhint %}
+
+### Prerequisite
+The cluster in which Argo CD apps exist should be added in **Global Configurations** → **Clusters and Environments**
+
+### Enabling ArgoCD App Listing
+
+{% embed url="https://www.youtube.com/watch?v=4KyYnsAEpqo" caption="Enabling External ArgoCD Listing" %}
+
+1. Go to the **Resource Browser** of Devtron.
+
+2. Select the cluster (in which your Argo CD app exists).
+
+3. Type `ConfigMap` in the 'Jump to Kind' field.
+
+4. Search for `dashboard-cm` using the available search bar and click it.
+
+5. Click **Edit Live Manifest**.
+
+6. Set the feature flag **ENABLE_EXTERNAL_ARGO_CD** to  **"true"**
+
+7. Click **Apply Changes**.
+
+8. Go back to the 'Jump to Kind' field and type `Pod`.
+
+9. Search for `dashboard` pod and use the kebab menu (3 vertical dots) to delete the pod.
+
+10. Go to **Applications** and refresh the page. A new tab named **ArgoCD Apps** will be visible.
+
+11. Select the cluster(s) from the dropdown to view the Argo CD apps available in the chosen cluster(s).
+
+    ![Figure 3: Cluster Selection for Argo CD Listing](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/argocd/argo-cluster-selection.jpg)
+
+---
+
+## View FluxCD App Listing
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need super-admin permission to view/enable/disable the FluxCD listing.
+{% endhint %}
+
+### Preface
+
+Devtron also supports the listing of [Flux CD](https://fluxcd.io/) apps in one dashboard, thus providing the same [advantages](#advantages) of [ArgoCD app listing](#view-argocd-app-listing).
+
+![Figure 4: FluxCD App List](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-listing.jpg)
+
+### Prerequisite
+The cluster in which Flux CD apps exist should be added in **Global Configurations** → **Clusters and Environments**
+
+### Enabling FluxCD App Listing
+
+{% hint style="info" %}
+### Tip
+You may refer the steps mentioned in the [Enabling ArgoCD App Listing](#enabling-argocd-app-listing) section since the procedure is similar. 
+{% endhint %}
+
+The feature flag to enable FluxCD app listing is **FEATURE_EXTERNAL_FLUX_CD_ENABLE** and it should be set to **"true"** as shown below.
```

Comment:
It should be like, here is the feature flag, refer how to add a feature flag in dashboard / orchestrator....